erpnext/maintenance/doctype/maintenance_schedule/maintenance_schedule.py

```diff
@@ -306,26 +305,28 @@ def get_pending_data(self, data_type, s_date=None, item_name=None):
 					return schedule.name
 
 @frappe.whitelist()
-def update_serial_nos(s_id):
-	serial_nos = frappe.db.get_value('Maintenance Schedule Detail', s_id, 'serial_no')
+def update_serial_nos(item_code, schedule=None):
```

Comment:
why is this called `update_serial_nos` when it just returns serial nos of an item from the Maintenance Schedule ?
`get_serial_nos_from_schedule` would be better ? or something like that